creator-node/.e2e-overrides/default-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "storagePath": "./file_storage",
+  "storagePath": "/file_storage",
```

Comment:
So with ./file_storage, we actually will re-use the same code dir file_storage folder for each instance, which makes tracking sync's etc. very dangerous. We either need to do this here OR we need to not mount the code repo (but that gives us hot reloading) cc @piazzatron

Reply:
Note: I also removed the ./file_storage folder which was comitted with a * .gitignore to keep the structure. I don't think it's a good idea for us to keep that structure as part of the repo, esp since we want to be running multiple of these on the same system sometimes

Reply:
Also cc @vicky-g  this explains what you were seeing with some files staying around even after pruning--they're probably in your local repo protocol/creator-node/file_storage